missing instructions for setting up a cross linker for macOS

add these instructions to the README: http://intermezzos.github.io/book/appendix/osx-install.html

`make cargo` on macOS gives this error. this is because macOS has the wrong `ld` (BSD), we need the GNU `ld`.

```
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-Tlayout.ld" "-Wl,-n" "-nostartfiles" "-L" "/Users/ag_dubs/.xargo/lib/rustlib/x86_64-unknown-intermezzos-gnu/lib" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/intermezzos.0.o" "-o" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/intermezzos" "-Wl,--gc-sections" "-nodefaultlibs" "-L" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/deps" "-L" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/build/intermezzos-491f39d1c9fb19a5/out" "-L" "/Users/ag_dubs/.xargo/lib/rustlib/x86_64-unknown-intermezzos-gnu/lib" "-Wl,-Bstatic" "-Wl,--whole-archive" "-l" "boot" "-Wl,--no-whole-archive" "-Wl,-Bdynamic" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/deps/libkernel_context.rlib" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/deps/libvga.rlib" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/deps/librlibc-975886925ea7e4d8.rlib" "/Users/ag_dubs/intermezzOS/kernel/src/target/x86_64-unknown-intermezzos-gnu/release/deps/libspin-d054d2d05cd86dc1.rlib" "/Users/ag_dubs/.xargo/lib/rustlib/x86_64-unknown-intermezzos-gnu/lib/libcore.rlib"
  = note: ld: unknown option: -n
clang: error: linker command failed with exit code 1 (use -v to see invocation)


error: aborting due to previous error

error: Could not compile `intermezzos`.

To learn more, run the command again with --verbose.
error: `cargo` process didn't exit successfully
make: *** [cargo] Error 101
```

--- a/src/README.md
+++ b/src/README.md
@@ -36,6 +36,17 @@ Then get this stuff:
 * `grub-mkrescue`: you may also need to install `xorriso`
 * `qemu-system-x86_64`
 
+If you're on OS X, you might want to use [this
+script](http://intermezzos.github.io/book/appendix/osx-install.html) to get
+them. You'll also need to set this in your `~/.cargo/config`:
+
+```toml
+[target.x86_64-unknown-intermezzos-gnu]
+linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
+```
+
+Where `yourusername` is your username.
+
 After all that setup, it’s as easy as:
 
 ```bash

